### ISSUE TITLE
commentFormatting warns in cgo on //line directive

### ISSUE THREAD

**Issue by jirfag** [go-critic/go-critic#755]
```
$ gocritic check -enable=commentFormatting ./test/testdata/cgo/main.go
/var/folders/nl/54f5x38s4m53mkzzj92zsj340000gn/T/gopackages354523912/go-build/command-line-arguments/main.cgo1.go:3:1: commentFormatting: put a space between `//` and comment text
```
Source file is [here](https://github.com/golangci/golangci-lint/blob/master/test/testdata/cgo/main.go).

There are 2 issues:
1. processing of cgo-compiled files and output of warnings from them. I don't known the proper way to handle it, in golangci-lint I have some hacks like [this](https://github.com/golangci/golangci-lint/blob/master/pkg/lint/astcache/astcache.go#L100) and [this](https://github.com/golangci/golangci-lint/blob/master/pkg/result/processors/cgo.go)
2. maybe we should add `//line` to exclude list? it doesn't complain about `//nolint` (but I didn't found this exclude list in code).

Reproduces with go1.10 but not with go1.11.

**Comment by quasilyte** [comment]
Fixed in 3c9d0fb535dd4f6cb7f4236ac1e39af5bf9d68a9
